lib/ex_ftx/orders/get_open_orders.ex

```diff
@@ -0,0 +1,30 @@
+defmodule ExFtx.Orders.GetOpenOrders do
```

Comment:
Can you rename this to `GetOrders`? My understanding is that it returns orders of any status. The docs are a bit misleading.

Reply:
done